OrbitCore/Params.cpp

```diff
@@ -70,11 +70,18 @@ void Params::Save() {
 
 //-----------------------------------------------------------------------------
 void Params::Load() {
-  std::ifstream file(Path::GetParamsFileName());
-  if (!file.fail()) {
+  std::string filename = Path::GetParamsFileName();
+  std::ifstream file(filename);
+  if (file.fail()) {
+    Save();
```

Comment:
Maybe add a short comment about why we want to save params if file does not exist.

Reply:
Done.